src/client/components/Dashboard.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import DeckItem from './DeckItem';
+// import _ from 'underscore';
+import $ from 'jquery';
```

Comment:
hmm... jQuery is a dependency for Materialize (if we use certain features). Use of jQuery here was to get something to work quickly. I agree, that it should be refactored at some point. I'll make a task for later to get this PR out.